src/routes/(site)/quiz/quiz.svelte

```diff
@@ -57,6 +57,7 @@
   }
 
   $: talliedResponses = (responses ?? []).reduce((tallies, match) => {
+    match = match?.toLocaleLowerCase();
```

Comment:
> toLocaleLowerCase() does not affect the value of the string itself. In most cases, this will produce the same result as [toLowerCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase), but for some locales, such as Turkish, whose case mappings do not follow the default case mappings in Unicode, there may be a different result.


In acmcsuf.com, we tend to use `toLowerCase` over `toLocaleLowerCase` since the method requires an argument to be of any difference to `toLowerCase`, which we practically never do.

```suggestion
    match = match?.toLowerCase();
```

Reply:
changed toLowerCase()
![acmlowercase](https://user-images.githubusercontent.com/54611122/200144598-67610aa0-96d0-4add-8571-ecf320484c46.png)

Reply:
This works for me. Resolved.